cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,26 @@
+class PaymentStepPage{
+
+    private bankWire: string;
+    private confirmPay: string;
```

Comment:
could you add to the attributes of each page object the type of element we are interacting with?